modules/dnn/src/ocl4dnn/include/default_kernel_config.hpp

```diff
@@ -140,77 +140,119 @@ NULL platform behavior
   clCreateContextFromType(NULL, CL_DEVICE_TYPE_CUSTOM)  No platform
   clCreateContextFromType(NULL, CL_DEVICE_TYPE_ALL)  No platform
 ********************************************************************************/
-"EU72_k1x1_cn512_g1_s1x1_d1x1_b1_in16x16_p0x0_num1_M112_activ1_eltwise0","7 2 8 2 1 1 8 1 0 ",
```

Comment:
Description above declares that OpenCL runtime is "r4.1.61547" - it is outdated / deprecated. Could you collect numbers with fresh Intel OpenCL NEO runtime?

Linux: https://github.com/intel/compute-runtime/releases
Windows: https://downloadcenter.intel.com/product/80939/Graphics-Drivers (or via regular Windows Update)

Reply:
The cache config is based on latest NEO runtime. The description info is also updated.